oojs.md

```diff
@@ -24,13 +36,24 @@ Create an `Album` class that inherits from `Playlist`. It should also...
 - Have an `artist` (String) property that is determined by some input
 
 ```js
-Your answer goes here
+let Album extends playlist {
+  constructor(artist, title, song) {
+    super(title, song);
+    artist = []
```

Comment:
- Need to say `this.artist =` so that it is attached to the `Album` instance that is being generated.
- `artist` needs to be populated by some user input (which needs to be a string, per the instructions). The way you have it now, it is always going to be initialized as an empty array.